components/brave_rewards/browser/database/database_contribution_info.cc

```diff
@@ -25,6 +25,25 @@ namespace {
 
     return std::stod(amount);
   }
+
+ledger::ReportType ConvertRewardsTypeToReportType(const int type) {
+  switch (type) {
+    case 2: {
+      return ledger::ReportType::AUTO_CONTRIBUTION;
```

Comment:
Why '2' and not the enum for 'RewardsType' - I thought we had one?

Reply:
we can't use ledger enums in switch

Reply:
Understood - in that case can you put the ledger 'type' associated with each value in as a comment alongside it (just for clarity)

Reply:
sorry, I didn't look at it closely. This is conversion from int that we get from DB into ledger enum, so we can't use anything as enum